xonsh/color_tools.py

```diff
@@ -397,17 +397,19 @@ def color_dist(x, y):
     return math.sqrt((x[0]-y[0])**2 + (x[1]-y[1])**2 + (x[2]-y[2])**2)
 
 
-def find_closest_color(x, pallette):
-    return min(sorted(pallette.keys())[::-1],
-               key=lambda k: color_dist(x, pallette[k]))
+def find_closest_color(x, palette):
+    return min(sorted(palette.keys())[::-1],
+               key=lambda k: color_dist(x, palette[k]))
 
 
-def make_pallete(strings):
-    """Makes a color pallete from a colection of strings."""
-    pallette = {}
+def make_palette(strings):
+    """Makes a color palette from a collection of strings."""
+    palette = {}
     for s in strings:
         while '#' in s:
             _, t = s.split('#', 1)
             t, _, s = t.partition(' ')
-            pallette[t] = rgb_to_ints(t)
-    return pallette
+            palette[t] = rgb_to_ints(t)
+    return palette
+
+make_pallette = make_palette
```

Comment:
The original function was misspelled as `make_pallete` (two Ls, one T).  So this should be `make_pallete` for backward compatibility?  Right now it's got two Ls _and_ two Ts.  Not sure whether the news entry has this same inconsistency.

Then again, I don't know anything about the PTK interface, so is `make_palette` a function we expect people to use directly?  If not, maybe we don't even have to bother with deprecating it (i.e., maybe we should just fix it here and not worry about keeping the typoed version around).  @astronouth7303, @laerus, @gforsyth, @scopatz, any thoughts?